changelog/2025.2.0.rst

```diff
@@ -57,12 +57,13 @@ Custom components were deprecated over a year ago and, as promised, support for
 encourage all developers to migrate their custom components to :doc:`/components/external_components`. Please see
 :ref:`a_note_about_custom_components` for a detailed explanation and next steps.
 
-Removal of ``armv7`` Docker Support
------------------------------------
+Removal of ``armv7`` Docker (and HA add-on) Support
+---------------------------------------------------
 
-ESPHome no longer supports ``armv7`` hardware. This is due to both waning support as it relates to tooling and
+ESPHome no longer supports ``armv7`` (32-bit ARM) hardware. This is due to both waning support as it relates to tooling and
```

Comment:
32-bits ARM is not just `armv7`, so this addition seem incorrect.

Reply:
Assuming we're not also providing `armhf`, then it's correct.  And it's more clear for users that don't understand what `armv7` is.  The 64-bit platforms actually have `64` in the name.

Reply:
Sure, but there are multiple 32 bits platforms for 32 bits.... 🤷

Reply:
But are we providing any of them?

Reply:
Maybe this?

```suggestion
ESPHome no longer supports the 32-bit  ``armv7`` hardware. This is due to both waning support as it relates to tooling and
```